toolkit/Makefile

```diff
@@ -77,6 +77,14 @@ ENABLE_CPU_PROFILE              ?= n
 ENABLE_MEM_PROFILE              ?= n
 ENABLE_TRACE                    ?= n
 
+# License checking tool
+##help:var:LICENSE_CHECK_DIRS:"<rpm_dir_1> <rpm_dir_2"=Space separated list of directories to recursively validate with 'license-check'. Default is RPMS_DIR.
+LICENSE_CHECK_DIRS ?= $(RPMS_DIR)
```

Comment:
Could you also add the documentation for the new tool into the readmes?

Reply:
Added to `1_initial_prep.md` with the other tools.
I expect most people will interact with this feature through the integration with the other build tools (scheduler, image fetcher?).